tests/test_plugins.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+from cms.api import add_plugin, create_page
+from cms.test_utils.testcases import CMSTestCase
+
+from djangocms_style.cms_plugins import StylePlugin
+
+
+class StyleTestCase(CMSTestCase):
+
+    def setUp(self):
+        self.language = "en"
+        self.home = create_page(
+            title="home",
+            template="page.html",
+            language=self.language,
+        )
+        self.home.publish(self.language)
+        self.page = create_page(
+            title="help",
+            template="page.html",
+            language=self.language,
+        )
+        self.page.publish(self.language)
+        self.placeholder = self.page.placeholders.get(slot="content")
+        self.superuser = self.get_superuser()
+
+    def tearDown(self):
+        self.page.delete()
+        self.home.delete()
+        self.superuser.delete()
+
+    def test_style_plugin(self):
+        plugin = add_plugin(
+            self.placeholder,
+            StylePlugin.__name__,
+            self.language,
+            label="Style plugin",
+            additional_classes="btn, btn-default",
+        )
+        plugin.clean()
+        self.assertEqual(plugin.plugin_type, "StylePlugin")
+
+    def test_plugin_structure(self):
+        request_url = self.page.get_absolute_url(self.language) + "?toolbar_off=true"
+
+        add_plugin(
+            self.placeholder,
+            StylePlugin.__name__,
+            self.language,
+            label="Style plugin",
+            additional_classes="btn, btn-default",
+        )
+        self.page.publish(self.language)
+
+        with self.login_user_context(self.superuser):
+            response = self.client.get(request_url)
+
+        self.assertContains(
+            response,
+            """<div class="container btn btn-default"></div>"""
+        )
```

Comment:
For this kind of assertions, it's best to use `assertContains([…], html=True)`. See https://docs.djangoproject.com/en/2.2/topics/testing/tools/#django.test.SimpleTestCase.assertContains